types/puppeteer/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for puppeteer 0.10
+// Type definitions for puppeteer 0.13
```

Comment:
v0.13 is not released yet, it should be 0.12 here

Reply:
Good catch, fixed.